samples/samples/autocommit.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import argparse
+
+from google.cloud.spanner_dbapi import connect
```

Comment:
As a heads up: this import will fail unless you put `spanner_dbapi` on the path yourself (calling `import pkg_resources` first will do it) until we fix the DBAPI packaging issues.